src/olympia/blocklist/tasks.py

```diff
@@ -121,14 +111,64 @@ def upload_filter(generation_time, is_base=True):
             server.publish_record(stash_upload_data)
             statsd.incr('blocklist.tasks.upload_filter.upload_stash')
 
+    oldest_base_filter_id = None
+
+    if len(filter_list) > 0:
+        for block_type in filter_list:
+            data = {
+                'key_format': MLBF.KEY_FORMAT,
+                'generation_time': generation_time,
+                'attachment_type': BLOCKLIST_RECORD_MLBF_BASE,
```

Comment:
For the soft block MLBF, this needs to be a different attachment type: `softblocks-bloomfilter-base` per https://searchfox.org/mozilla-central/rev/f732a1651018b7c32002981a9c8b8613975ffbf9/toolkit/mozapps/extensions/Blocklist.sys.mjs#908

Reply:
What do we expect for the filename.. is it always `filter.bin` or should be based on the block_type? Currently it is different for each blocktype, but not sure the expectations.

Reply:
The file name does not matter, the attachment type does, however.